content/embeds/redis-feature-sets.md

```diff
@@ -0,0 +1,9 @@
+A Redis feature set includes a specific Redis database version along with the advanced capabilities and data structures provided by compatible module versions.
```

Comment:
Just a caveat here: Redis CE doesn't include the advanced capabilities by default. Customers will either have to download and install them separately, or use Redis Stack. This, of course, will go away when CE8 ships, essentially replacing both standalone Redis CE and Redis Stack.

So, really, this just applies to RS, RC, and Redis Stack.

Reply:
Correct.  We can add that in if you like.  I don't think users are consuming modules separately often.  We see in the field customers running redis-stack in production